spec/amp-story-player.md

```diff
@@ -119,29 +117,14 @@ The `<amp-story-player>` component contains one or more `<a>` tags. Point the hr
 
 Place the story's title within the `<a>` tag. This provides a better user experience and allows search engines to crawl embedded stories.
 
-Use a poster image as a placeholder to display to users while the story loads. Add the `--story-player-poster` CSS variable as an inline style of the `<a>` tag and point to the poster image URL.
+Use a poster image as a placeholder to display to users while the story loads. To do this, add an image tag as a child of the `<a>` tag with the following configuration: `<img src= "" loading="lazy" width="100%" height="100%" data-amp-story-player-poster-img></img>`.
```

Comment:
I don't think "100%" is a valid value for IMG tag width/height. 

https://www.w3schools.com/tags/tag_img.asp:
> Value: pixels

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img
> The intrinsic height of the image, in pixels. Must be an integer without a unit.